ui/src/app/store/types/node.ts

```diff
@@ -77,6 +104,37 @@ export enum NodeActions {
   UNLOCK = "unlock",
 }
 
+export enum TestStatusStatus {
```

Comment:
Even though this seems to just be a subset of `ScriptResultStatus`, it's actually derived in a completely different way so I think it makes sense for this to be a different enum.